src/Checkbox/Input.js

```diff
@@ -62,13 +56,11 @@ Input.propTypes = {
     onChange: propTypes.func.isRequired,
     name: propTypes.string.isRequired,
 
-    autocomplete: propTypes.string,
     tabIndex: propTypes.string,
 
     onFocus: propTypes.func,
     onBlur: propTypes.func,
     checked: propTypes.bool,
     disabled: propTypes.bool,
-    autofocus: propTypes.bool,
-    readonly: propTypes.bool,
+    focus: propTypes.bool,
```

Comment:
`focus` is used in `componentDidMount` and should be declared.

Reply:
Well spotted. I did a commit that adds this to Icon too.

Reply:
Thanks. 👍